IREC/IREC_PDD.tex

```diff
@@ -0,0 +1,383 @@
+
+%\begin{figure}
+  %\includegraphics[width=\linewidth]{figs/project-life-cycle.png}
+  %\caption{A PDD is the first piece of documentation to be archived in the project life cycle. Since the life cycle can be iterative, a new design document may also refer to one or more previous SPPs.}
+%\label{fig:lifecycle}
+%\end{figure}
+
+% \section{Secondary Objectives}
+% \label{sec:secondary-obj}
+% Secondary Objectives are lower priority or bonus objectives that are significant but not the main focus of the project. This template does not have secondary objectives.
+
+\section{Benefit to SPEX}
+\label{sec:Benefit to SPEX}
+% One of the core values of SPEX is to provide opportunities for academic and professional growth for its members,
+% and to challenge them with interesting projects.
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+SPEX's involvement in this globally-identified competition will serve to provide exposure and merit to the team and any sponsors involved in contributing.
+Additionally, individual students engaged in this endeavor will also benefit by improving their technical knowledge and abilities while
+participating in a major event which can also be used to improve their professional skillset.
+
+% Below I have used subsections to identify key ideas in this section. These particular subsections are not required as part of the SPEX Standard, but serve as an example of using subsections in a text
+
+\subsection{Benefit to RIT}
+\label{subsec:Benefit to RIT}
+By attending the event and competing against top universities, RIT will be able to use this as an opportunity to showcase
+how passionate and involved their students are in space exploration. This involvement can be used in media as a tool for
+achieving more funds and attracting new students to the University. Since RIT has not been involved with IREC before, SPEX's attendance at
+this competition can also show how the University and its students are growing and taking on new challenges.
+
+\subsection{Partnership with Launch Initiative}
+\label{subsec:Partnership with Launch Initiative}
+Since the founding of SPEX, there has never been a strong relationship with the Launch Initiative team. By cooperatively participating
+in this event, a great opportunity is presented to create a good relationship between two of the largest space-related teams on
+campus. In the future, united efforts between the two teams could result in more advanced projects which will provide
+even more exposure to both RIT space teams.
+
+\section{Implementation}
+\label{sec:Implementation}
+
+\subsection{Selection}
+\label{subsec:Selection}
+  % What path do you anticipate the project to take?
+A decision matrix was created which accounts for the following factors and assigned a maximum possible
+score based on importance to the selection process:
+\begin{itemize}
```

Comment:
I hate bulleted lists! They're ugly and not the most efficient way to convey information. Consider using a table.